src/sdk/object_sdk.js

```diff
@@ -94,6 +108,38 @@ class ObjectSDK {
         return this._setup_bucket_namespace(bucket);
     }
 
+    // TODO: Handle accountspace operations / RPC auth (i.e system, account, anonymous) and anonymous access
+    async authorize_request_account({ bucket, op_name }) {
+        const ns = bucket && await this.read_bucket_sdk_namespace_info(bucket);
+        const token = this.get_auth_token();
+        if (!token) return;
+        if (ns) return 'NotImplemented';
```

Comment:
I don't understand this condition. 
Even if it's correct it is so counter-intuitive that it can't be written without a comment.